job_dsl/standard_job_checkmarx.groovy

```diff
@@ -0,0 +1,98 @@
+// Can't use env.FOO = {FOO} to transfer JJB vars to groovy
```

Comment:
Yes, you can. You're doing in this pull request with the inject property.

Reply:
The comment is trying to say that JJB macros will not be expanded, which is correct, because job_dsl/* is not processed by JJB.